task-git.sh

```diff
@@ -3,9 +3,9 @@
 # Get task command
 TASK_COMMAND="task ${@}"
 # Get data dir
-DATA_RC=$(task _show | grep data.location)
+DATA_RC=$(/usr/bin/task _show | grep data.location)
```

Comment:
Not sure about this line as this would break for me, my `task` is at `/usr/local/bin/task`.